src/swagger-specs/query-service.yaml

```diff
@@ -1825,4 +2174,544 @@ definitions:
+            example: 'true'
+          emailNotifications:
+            type: boolean 
+            description: A true value indicates that the users want to receive an alert via email. A false value would not provide an alert via email.
+            example: 'true'
+            
+  emailIdForSubscribing:
+    type: string
+    description: A list of Adobe registered emailIDs that want to subscribe to the alert for a given assetId and alertType.
+    example: 'rrunner@adobe.com'
+
+  alert_subscription_response_body:
+    type: object
+    description: The response when subscribing emailIDs to an alert for an assetId and alertType.
+    required:
+      - assetId
+      - alertType
+      - subscriptions
+      - _links
+    properties:
+      id:
+        type: string
+        description: The alert name that appears in the alerts tab of the Adobe Experience Platform UI.
+        example: 'a679dd0e-bcb2-4e69-a610-22d17ba98cac'
+      assetId:
+        type: string
+        description: The query or schedule ID that the alert is created for and subscribed to.
+        example: '4422fc69-eaa7-464e-945b-63cfd435d3d1'
+      alertType:
+        type: string
+        description: This property determines the state of query execution that triggers a user to receive an alert. The alert type is one of three values (start, success, failure).
+        example: 'start'
+      status:
+        $ref: '#/definitions/Status'
+      subscriptions:
+        type: object
+        required:
+          - emailIds
+          - inContextNotifications
+          - emailNotifications
+        description: This represents every emailId that has subscribed to the alert and what notification channels the emailId's have subscribed to.
+        $ref: '#/definitions/Subscriptions'
+      _links:
+        $ref: '#/definitions/alert_links'
+
+        # properties:
+        #   emailIds:
+        #     type: array
+        #     description: These are Adobe registered email addresses. The maximum number of emailIDs that can be registered at once is five.
+        #     items:
+        #       $ref: '#/definitions/emailIdForSubscribing'
+        #   inContextNotifications:
+        #     type: boolean
+        #     description: A true value represents the users with an emailID in the list that want to receive an alert through the Experience Platform UI. A false value would not provide an alert in the Adobe Experience Platform UI.
+        #     example: 'false'
+        #   emailNotifications:
+        #     type: boolean
+        #     description: A true value represents the users with an emailID in the list that want to receive an alert via email. A false value would not provide an alert via email.
+        #     example: 'true'
+
+  alert_links:
+    type: object
+    description: Useful information including the URIs and methods for common alert subscriptions use cases.
```

Comment:
Seeing a lot of examples of this -- please avoid "Useful information" as a description. Whether or not the information is "useful" for a user is entirely dependent on context that we don't necessarily have.

Just describe this as an object and the type of information it contains without the qualifier.